src/ImageSharp.Web.Providers.Azure/Resolvers/AzureBlobStorageImageResolver.cs

```diff
@@ -16,22 +15,13 @@ namespace SixLabors.ImageSharp.Web.Resolvers.Azure
     /// </summary>
     public class AzureBlobStorageImageResolver : IImageResolver
```

Comment:
Code coverage is reporting a delta of -12.31% for this file, however coverage is 100%. I've read their docs, I don't get it.